File: dataset/human/java/file_2617.java
import java.util.*;
public class Main {
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);

		// 文字列の入力
		String[] s = sc.next().split("");
        Arrays.sort(s, Comparator.naturalOrder());
        String sortedS;
        String sortedT;
        StringBuilder builder = new StringBuilder();
        for(String value : s) {
          builder.append(value);
        }
        sortedS = builder.toString();
        builder.setLength(0);
        String[] t = sc.next().split("");
        Arrays.sort(t, Comparator.reverseOrder());
        for(String value : t) {
          builder.append(value);
        }
        sortedT = builder.toString();

        String[] list = {sortedS, sortedT};
        Arrays.sort(list, Comparator.naturalOrder());
    
		// 出力
		System.out.println(!sortedS.equals(sortedT) && list[0].equals(sortedS) ? "Yes" : "No");
	}
}

File: dataset/human/c/file_2617.c
#include<stdio.h>
int main(){
  char s[4];
  scanf("%s",s);
  if((s[0]-'a')+(s[1]-'a')+(s[2]-'a')==3 &&
     (s[0]-'a'+1)*(s[1]-'a'+1)*(s[2]-'a'+1)==6)
    printf("Yes");
  else
    printf("No");
  return 0;
}
